### DISCUSSION TITLE
Поддерживается создание навыков умного дома?

### DISCUSSION THREAD

**Discussion by mikhail-rozum** [alexvolchetsky/yandex.alice.sdk#4] [General]
Добрый день! Поддерживает ли библиотека создание навыков умного дома? Если да, есть пример?

**Comment by valchetski** [comment]
Здравствуйте! К сожалению, в данный момент нет. Но могу попробовать реализовать поддержку в будущих версиях

**Comment by valchetski** [comment]
Поддержка навыков умного дома добавлена. Больше информации в релизе: https://github.com/alexvolchetsky/yandex.alice.sdk/releases/tag/v2.1.0
